build/dependencies.props

```diff
@@ -0,0 +1,24 @@
+<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  <PropertyGroup Label="Package Versions">
+    <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15549</InternalAspNetCoreSdkPackageVersion>
```

Comment:
@natemcmaster Not sure if this has been covered by email or in a brownbag or something, but could you briefly describe how these version numbers are going to get updated? Who does it, using what tool, and how often should we expect the updates to be applied?

Reply:
It was in an email to the runtime team. The details are also posted here https://github.com/aspnet/KoreBuild/issues/239#issuecomment-340884986.

Short version:
`run.ps1 upgrade deps` will update the file to latest. Our CI will run it once a week on repos, but you're welcome to do it on your own and merge the results if you need newer dependencies.

Reply:
Sounds good - thanks for the info!